### DISCUSSION TITLE
DISABLE combat positioning

### DISCUSSION THREAD

**Discussion by xBarselonx** [trickerer/Trinity-Bots#304] [Q&A]
Привет!
trickerer, недавно заметил, что в меню сплетен ботов появилась новая функция «DISABLE combat positioning». Не совсем понял как она работает. Объясни пожалуйста на примере как это работает с рдд и мдд?

**Comment by trickerer** [comment]
Это было предложено в #271 (3)
В README описано в общих чертах: https://github.com/trickerer/Trinity-Bots/#npcbot-formation

Дальние боты не будут пытаться следовать за целью. Совсем. Вместо этого они сохранят режим следования за игроком, вот и всё
